src/main/resources/application.yml

```diff
@@ -5,11 +5,13 @@ spring:
     password: postgres
     driverClassName: org.postgresql.Driver
     initialization-mode: always
+    hikari:
+      maximumPoolSize: 50
 
   jpa:
     database-platform: org.hibernate.dialect.PostgreSQL94Dialect
     hibernate:
-      ddl-auto: create
+      ddl-auto: update
```

Comment:
Minor comment but my `SampleReceiverIT` is failing its second run due to a key constraint error:
```
2019-04-26 15:37:22.782 ERROR 78948 --- [           main] o.h.engine.jdbc.spi.SqlExceptionHelper   : ERROR: update or delete on table "cases" violates foreign key constraint "fkgqdn6408vhuo465xp8tlibp7c" on table "uac_qid_link"
  Detail: Key (case_ref)=(10000051) is still referenced from table "uac_qid_link".
```
I don't know enough about Hibernate, but would we add a `ddl-auto: create` into the test config to get around this?

Reply:
Fixed now